git-remote-mediawiki

```diff
@@ -104,7 +104,7 @@ if (!$fetch_strategy) {
 	$fetch_strategy = run_git('config --get mediawiki.fetchStrategy');
 }
 chomp($fetch_strategy);
-if (!$fetch_strategy) {
+if (!$fetch_strategy || $#tracked_pages > -1 ) {
```

Comment:
This is often the right thing to do, but is it always the case?

If `tracked_pages` contains a large subset of the wiki, one may still want to fetch by revision and filter only revisions that touch the pages we want. Or is the "fetch by revision and filter only revisions touching tracked pages" behavior broken? If so, this deserves a comment explaining why we fall-back to a safe behavior.